cmd/wallet-web/src/components/Vaults/AddVaultModal.vue

```diff
@@ -5,104 +5,56 @@
 -->
 
 <template>
-  <modal>
-    <div
-      class="
-        flex
-        overflow-y-auto
-        fixed
-        inset-0
-        z-50
-        justify-center
-        items-center
-        bg-neutrals-dark bg-opacity-50
-      "
-    >
+  <modal :show="showModal">
+    <template #content>
+      <!--content-->
```

Comment:
Maybe we just keep these comments in the Modal itself and remove from here, the template name is self-explanatory